lib/jekyll/tidy.rb

```diff
@@ -6,55 +6,59 @@
 
 module Jekyll
   module Tidy
-    def self.init(site)
-      @JEKYLL_CONFIG = site.config
-    end
+    class << self
+      def init(site)
+        @jekyll_config = site.config
+      end
 
-    def self.jekyll_config
-      @JEKYLL_CONFIG || Jekyll.configuration({})
-    end
+      def jekyll_config
+        @jekyll_config || Jekyll.configuration({})
+      end
 
-    def self.exclude?(path, override = {})
-      config = jekyll_config.merge(override)
-      exclude_paths = config["jekyll_tidy"] && config["jekyll_tidy"]["exclude"]
-      exclude_paths.to_a.any? { |exclude| File.fnmatch(exclude, path) }
-    end
+      def exclude?(path, override = {})
+        config = jekyll_config.merge(override)
+        exclude_paths = config["jekyll_tidy"] && config["jekyll_tidy"]["exclude"]
+        exclude_paths.to_a.any? { |exclude| File.fnmatch(exclude, path) }
+      end
 
-    def self.compress_output?
-      jekyll_config["jekyll_tidy"] && jekyll_config["jekyll_tidy"]["compress_html"]
-    end
+      def compress_output?
+        jekyll_config["jekyll_tidy"] && jekyll_config["jekyll_tidy"]["compress_html"]
+      end
 
-    def self.output_clean(output, compress = false)
-      if compress
-        return HtmlCompressor::Compressor.new.compress output
-      else
-        return HtmlBeautifier.beautify output
+      def output_clean(output, compress = false)
+        if compress
+          HtmlCompressor::Compressor.new.compress output
+        else
+          HtmlBeautifier.beautify output
```

Comment:
In these particular cases, where we have multiple return paths depending on conditional branching, we prefer explicit `return` statements to avoid confusion.